app/models/jobs-model.js

```diff
@@ -9,16 +10,25 @@ const sortRef = {
   employer: { field: 'employer', direction: 'ASC' },
 };
 
+const historicDate = {
+  week: moment().subtract(moment.duration(1, 'w')),
+  fortnight: moment().subtract(moment.duration(2, 'w')),
+  month: moment().subtract(moment.duration(1, 'M')),
+  none: moment('2016-01-01'),
+};
+
 module.exports = db.Model.extend(
   {
     tableName: 'jobs',
     hasTimestamps: true,
   },
   {
-    findAllByAccountId(accountId, sort) {
+    findAllByAccountId(accountId, sort, filter) {
       const sortOrDefault = (sort == null || sort === '') ? 'created' : sort;
+      const filterOrDefault = (filter == null || filter === '') ? 'none' : filter;
```

Comment:
Hmmmm read around this quite a bit previously and came to this solution.  If the link you provided was right you would just have to check:

    findAllByAccountId(accountId, sort, filter) {
      const sortOrDefault = sort  ? 'created' : sort;
      const filterOrDefault = filter ? 'none' : filter;

... 20 broken tests later i'm not sure if that would facilitate empty entries in the possible URL's e.g.
http://localhost:3000/42c28b92-3533-4823-ade0-a1371ad83119?sort=&filter=